src/phoenix/db/models.py

```diff
@@ -87,6 +91,7 @@ class Span(Base):
     status: Mapped[str]
     status_message: Mapped[str]
 
+    # TODO(mikeldking): is computed columns possible here
```

Comment:
maybe a [composite](https://docs.sqlalchemy.org/en/20/orm/composites.html) column

Reply:
let's look into it.